src/App.test.js

```diff
@@ -715,4 +716,50 @@ describe('App component', () => {
       expect(localStorageItems['dopplerFirstOrigin.value']).toBeUndefined();
     });
   });
+
+  describe('activation in progress parameter', () => {
+    it('should call google adwords', () => {
+      // Arrange
+      const dependencies = {
+        window: {
+          gtag: jest.fn(),
+          Menubutton: MenubuttonFake,
```

Comment:
Is this line required?

Reply:
yes, because use in `LanguageSelector` component